lib/services/rally/rally_hierarchical_requirement_resource.rb

```diff
@@ -200,6 +202,57 @@ def map_requirement(parent_id, release_id, aha_requirement)
     attributes
   end
 
+  def maybe_add_tags_to_object(attributes, aha_object)
+    if @service.data.send_tags == "1"
+      object_tags = aha_object.tags
+      attributes[:Tags] = [] if object_tags
+      attributes[:Tags] = get_or_create_tag_references(object_tags) if object_tags && !object_tags.empty?
+    end
+  rescue AhaService::RemoteError => e
+    logger.error("Failed to create tag #{tag_name}: #{e.message}")
+  end
+
+  def get_or_create_tag_references(tags)
+    # Rally doesn't want escaped quotes or parens
+    query_params = build_tag_query(tags).gsub(" ", "%20").gsub("=", "%3D").gsub("&", "%26")
```

Comment:
CGI escape escapes parens, and Rally does not accept the query if the parens are escaped.